ast/ddl.go

```diff
@@ -1838,6 +1838,8 @@ const (
 	AlterTableRemovePartitioning
 	AlterTableWithValidation
 	AlterTableWithoutValidation
+	AlterTableSecondaryLoad
+	AlterTableSecondaryUnLoad
```

Comment:
... but I'd appreciate if you spell it as 

```suggestion
	AlterTableSecondaryUnload
```

Reply:
ok